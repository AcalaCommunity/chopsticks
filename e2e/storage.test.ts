import { describe, it } from 'vitest'

import { api, expectJson } from './helper'

describe('storage', () => {
  it('getStorage', async () => {
    await expectJson(api.query.timestamp.now()).toMatchInlineSnapshot('1666661202090')
    await expectJson(
      api.query.tokens.accounts('5F98oWfz2r5rcRVnP9VCndg33DAAsky3iuoBSpaPUbgN9AJn', { Token: 'AUSD' })
    ).toMatchSnapshot()
  })
})
