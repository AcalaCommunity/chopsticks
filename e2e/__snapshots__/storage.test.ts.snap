// Vitest Snapshot v1

exports[`storage > getStorage 2`] = `
{
  "free": "0x0000000000000000016e0dc6c80c5343",
  "frozen": 0,
  "reserved": 0,
}
`;
